src/licenses/licensesClient.ts

```diff
@@ -617,4 +577,99 @@ export class LicensesClient extends AbstractRestfulClient {
 
     return await this.put(payload, parameters);
   }
+  private createFilters(
+    parameters: string | BaseParameters<unknown, unknown>,
+    keyParent: string,
```

Comment:
Create a new type `KeyParent`:
```ts
type KeyParent = 'license' | 'offer';
```
```suggestion
    keyParent: KeyParent,
```